src/styles/style.css

```diff
@@ -0,0 +1,250 @@
+body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__logo-img {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+  font-style: normal;
+  line-height: 60px;
+  display: flex;
+  align-items: center;
+  position: relative;
+}
+
+.is-active,
+.nav__link[data-qa='nav-hover']:hover {
```

Comment:
The selector `.is-active` is used multiple times in the stylesheet. Consider using a more specific class name to avoid potential conflicts or unintended styling.